### ISSUE TITLE
Advanced search: methods ontology 

### ISSUE THREAD

**Issue by HSimek** [ComPlat/chemotion_ELN#2263]
Search according to ´methods ontology´ works, but you can only select a type from continuant fiat boundary, process or recently selected. 
![type search](https://github.com/user-attachments/assets/921f897b-9ca6-4aed-92db-1145b3677339)

I am not able to enter text in this field on my own, so I cannot search for e.g. HSQC if it is not in the list of recently selected. Furthermore, I can not open the list of recently selected items by clicking on ´recently selected´, I have to click on the arrow before the text (I think this might confuse users and lead them to believe that there is no entry there)”

**Comment by nicolejung** [comment]
I checked the issue for the versions 1.10.3 and 1.10.4 and it worked for me - could you check again please?

**Comment by nicolejung** [comment]
You are right, the advanced search does not contain the option to add text as input for the search in the ontology terms ->needs discussion in the development team
